courses/free-courses-id.md

```diff
@@ -102,7 +103,10 @@
 
 * [Belajar Fundamental Aplikasi Flutter](https://www.dicoding.com/academies/195) - Dicoding, membutuhkan registrasi
 * [Belajar Membuat Aplikasi Flutter untuk Pemula](https://www.dicoding.com/academies/159) - Dicoding, membutuhkan registrasi
-* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana?thumbnail=nk4neM2UyG.110&main_leads=browse) - BuildWithAngga
+* [Flutter Basic State Management](https://buildwithangga.com/kelas/flutter-developer-basic-state-management) - BuildWithAngga
+* [Flutter for Designer](https://buildwithangga.com/kelas/flutter-for-designer-design-to-code) - BuildWithAngga
+* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana) - BuildWithAngga
+* [Flutter Mobile Apps](https://buildwithangga.com/kelas/flutter-mobile-apps) - BuildWithAngga
```

Comment:
- L106: wrong resource title
- Creators: Rifqi Eka, Tasya Agnes, Angga Risky
- Platform: BuildWithAngga
- Notes: requires sign-up

Please, use this snippet as fix:

```markdown
* [Flutter Developer: Basic State Management](https://buildwithangga.com/kelas/flutter-developer-basic-state-management) - Tasya Agnes (BuildWithAngga) *(phone number and email address required)*
* [Flutter for Designer](https://buildwithangga.com/kelas/flutter-for-designer-design-to-code) - Rifqi Eka (BuildWithAngga) *(phone number and email address required)*
* [Flutter Membangun Website](https://buildwithangga.com/kelas/flutter-membangun-website-sederhana) - Rifqi Eka (BuildWithAngga) *(phone number and email address required)*
* [Flutter Mobile Apps](https://buildwithangga.com/kelas/flutter-mobile-apps) - Angga Risky (BuildWithAngga) *(phone number and email address required)*
* ```